pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -251,6 +251,7 @@ private static long getRandomInitialDelayInSeconds() {
   private static final String DEFAULT_DIM_TABLE_MAX_SIZE = "200M";
 
   private static final String DEFAULT_PINOT_FS_FACTORY_CLASS_LOCAL = LocalPinotFS.class.getName();
+  public static final String DISABLE_GROOVY = "controller.disable.groovy";
```

Comment:
I noticed that the configs defined in this file don't start with the `pinot.` prefix like how the properties are defined in CommonConstants. Should we make this consistent with the broker config i.e. `pinot.broker.disable.groovy`?

Reply:
This is a legacy issue. Let's keep it the current way, and we can fix it all together in the future.